specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -0,0 +1,2958 @@
+          {
+            "in": "query",
+            "name": "lastUpdatedEndTime",
+            "description": "End DateTime(ISO 8601 literal format) of the last updated time range to filter tests.",
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "in": "query",
+            "name": "continuationToken",
+            "description": "Continuation token to get the next page of response",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "maxPageSize",
+            "description": "Number of results in response.",
+            "type": "integer",
+            "format": "int32",
+            "default": 50
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The requested tests.",
+            "schema": {
+              "$ref": "#/definitions/TestModel20221101ResourceList"
+            }
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Search test with filters": {
+            "$ref": "./examples/ListLoadTestSearch.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/loadtests/{testId}/files/{fileId}": {
+      "put": {
+        "tags": [
+          "Test"
+        ],
+        "summary": "Upload input file for a given test name. File size can't be more than 50 MB. Existing file with same name for the given test will be overwritten. File should be provided in the request body as multipart/form-data.",
+        "operationId": "Test_UploadFile",
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/TestIdPathParameter"
+          },
+          {
+            "in": "path",
+            "name": "fileId",
+            "description": "Unique name for test file, must contain only lower-case alphabetic, numeric, underscore or hyphen characters.",
+            "required": true,
+            "type": "string",
+            "maxLength": 50,
+            "minLength": 2,
+            "pattern": "^[a-z0-9_-]*$"
+          },
+          {
+            "in": "query",
+            "name": "fileType",
+            "type": "string",
+            "enum": [
+              "JMX_FILE",
+              "USER_PROPERTIES",
+              "ADDITIONAL_ARTIFACTS"
+            ],
+            "x-ms-enum": {
```

Comment:
Please add descriptions for all values in the x-ms-enum extension. This will benefit the generated REST API docs.